test/functional/setup.js

```diff
@@ -71,8 +71,20 @@ function openRemoteBrowsers () {
             };
 
             var openBrowserPromises = browsersInfo.map(function (browserInfo) {
-                return connector.startBrowser(browserInfo.settings, browserInfo.connection.url, buildInfo,
-                    isBrowserStack ? { openingTimeout: BROWSER_OPENING_TIMEOUT } : null);
+                if (!isBrowserStack || browserInfo.settings.alias !== 'chrome-osx') {
```

Comment:
Please add a note that it's a temporary workaround to fix a problem on BrowserStack when a new page in a new tab is opened during testing unexpectedly